catalogue/webapp/pages/works.js

```diff
@@ -440,7 +441,7 @@ Works.getInitialProps = async (ctx: Context): Promise<Props> => {
   const worksOrError = shouldGetWorks
     ? await getWorks({
         params: apiProps,
-        pageSize: isImageSearch ? 24 : undefined,
+        pageSize: isImageSearch ? 100 : undefined,
```

Comment:
We are loading these in an enhanced way yes?

Reply:
Although not if you don't have JS - I wouldn't be happy getting 100 images if I didn't have JS on.

Reply:
That's a good point. Will update.

Reply:
Short of doing this image search entirely client-side, I'm not sure how this would work.

I take your point that we don't want to bombard people with unnecessary images, but if I'd actively selected 'images only', I think I'd expect to see images.

Reply:
Maybe not load in a 100 server side?

Reply:
something like
```
getWorks({
    params: apiProps,
    pageSize: isServer ? undefined : isImageSearch ? 100 : undefined,
})
```
?

Reply:
<img width="756" alt="Screenshot 2020-02-13 at 15 53 07" src="https://user-images.githubusercontent.com/31692/74452474-0a5b8200-4e79-11ea-8f20-341532e15856.png">

They load in 4 more than that.

Reply:
How to rehydrate to 100 once we know you've got JS though? Think I'll take this out of this PR and add a `TODO`